.github/ISSUE_TEMPLATE/feature_request.md

```diff
@@ -12,13 +12,29 @@ assignees: ''
 **Is your feature request related to a problem? Please describe.**
 A clear and concise description of what the problem is.
```

Comment:
please move this to be after `How it works Now`

Reply:
Done